AppCenter/AppCenter/Internals/Util/MSEncrypter.m

```diff
@@ -171,4 +217,22 @@ + (NSData *)generateKeyWithTag:(NSString *)keyTag {
   return resultKey;
 }
 
++ (NSData *)generateInitializationVector {
+  uint8_t *ivBytes = malloc(kCCBlockSizeAES128 * sizeof(uint8_t));
+  memset((void *)ivBytes, 0x0, kCCBlockSizeAES128);
```

Comment:
Just curiosity, any reason of initialize the memory space?

Reply:
Nope. This is copied from the code to generate key, which was doing this. We didn't figure out why, though. If nobody has a reason for it, we can just remove it from both places.